android/app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -704,6 +704,10 @@
 	<string name="enable_keep_screen_on">保持螢幕開啟</string>
 	<!-- Description in preferences -->
 	<string name="enable_keep_screen_on_description">啟用後，顯示地圖時畫面將始終開啟。</string>
+	<!-- A preference title; keep short! -->
+	<string name="enable_show_on_lock_screen">在鎖定螢幕上顯示</string>
```

Comment:
@zyphlar it should be in data/strings/strings.txt

Reply:
Hmm somehow Github search didn't show this string as existing in strings.txt. I'll add.

Reply:
@biodranik is this better?